code/DDSCodeTester.cpp

```diff
@@ -2936,6 +2936,15 @@ void dds_qos_examples()
         consistency_qos.type_consistency.m_kind = TypeConsistencyKind::ALLOW_TYPE_COERCION;
         //!--
     }
+
+    {
+        //DDS_QOS_POLICY_COUNT_SEQ
+        RequestedIncompatibleQosStatus status;
+
+        // Set the count for ReliabilityQosPolicy
+        status.policies[RELIABILITY_QOS_POLICY_ID].count = 4;
```

Comment:
Won't a user be more interested in reading this value instead of setting it? Why would a user do this?

Reply:
You are right, how about:
```
DataReader* data_reader =
        subscriber->create_datareader(topic, DATAREADER_QOS_DEFAULT);

// Get how many times ReliabilityQosPolicy was not compatible with a remote writer
RequestedIncompatibleQosStatus** status = data_reader->get_requested_incompatible_qos_status();
uint32_t incompatible_reliability = status.policies[RELIABILITY_QOS_POLICY_ID].count;
```

Reply:
That looks good :+1: Small suggestion:
```diff
- uint32_t incompatible_reliability = status.policies[RELIABILITY_QOS_POLICY_ID].count;
+ uint32_t incompatible_reliability_count = status.policies[RELIABILITY_QOS_POLICY_ID].count;
```